### DISCUSSION TITLE
Where could I find document for older version of emscripten?

### DISCUSSION THREAD

**Discussion by Thaina** [emscripten-core/emscripten#19123] [Q&A]
I am using emscripten only from unity, which each version stick with specific version of unity (2021.3 using emscripten 2.0.19)

And so I am very needed to read document of that specific version. And for each unity version I use I need to find the exact version of emscripten document

And so are there any url path that emscripten website are using to revert the site to specific version? Are there any method more convenient than build my own site?

If there is no existing way, could you please consider to have version selector built in to the docs?

**Comment by sbc100** [comment]
Think we only have a single version of the docs published at a given time.   I don't see why we couldn't have multiple versions published though.   Perhaps we can start by published the 2.0.19 version of the docs in a subdirectory of https://emscripten.org/docs/?    @kripken WDYT?

**Reply by kripken** [reply]
Another option might be to build&bundle the docs in each build perhaps? I'm not sure what's best between those options but I've seen projects do both.

**Reply by sbc100** [reply]
Publishing docs for every release would make the release process a fair bit more complex.. so if we do decide to do that we should make sure it automatic in some way (perhaps using a github action to do the publication).

**Reply by Thaina** [reply]
> Another option might be to build&bundle the docs in each build perhaps? I'm not sure what's best between those options but I've seen projects do both.

Being a website is the most convenient place to read doc. It could also be indexed with google and everyone already know its there

Also the approach to stick the doc with the build cannot solve my problem that I need to read the 2.0.19 doc for using with unity, which already package emscripten and will not have doc there


ps. If emscripten using (or switching) to firebase hosting, there is a channel feature that allow multiple version with subdomain https://firebase.google.com/docs/hosting/manage-hosting-resources
